packages/tss/test/helpers.ts

```diff
@@ -72,6 +72,9 @@ export async function assignTssDkgKeys(opts: {
 }) {
   const { serviceProvider, verifierName: verifier, verifierId } = opts;
 
+  serviceProvider.verifierName = verifier;
+  serviceProvider.verifierId = verifierId;
```

Comment:
why you need to add this?

Reply:
I think it was added during writing test and some sp is missing verifer and verifierId, and getTssPubKey is throwing error.
Seems like current set of tests's sp do not need the assignment.
will remove that